src/index.js

```diff
@@ -23,10 +23,10 @@ function getLatestVersion() {
 
 function replaceMocha(str) {
   return str
-    .replace('--compilers js:babel-core/register', '--compilers js:@babel/register')
-    .replace('--compilers js:babel-register', '--compilers js:@babel/register')
-    .replace('--require babel-register', '--require @babel/register')
-    .replace('--require babel-polyfill', '--require @babel/polyfill');
+    .replace(/--compilers +js:babel-core\/register/, '--compilers js:@babel/register')
```

Comment:
`--compilers\s+js:babel-core\/register`

If you want to account for any amount of whitespace.

Reply:
I think both ok.